updated navbar, created form for seller and also updated footer section

Aradhya-005 opened the pull request:
This pull request includes the following updates:

Updated the navbar:
Implemented changes to enhance navigation experience.

Created a form for seller:
Designed a user-friendly and responsive form for sellers to input their information.

Updated footer section:
Enhanced the layout for better readability.

These changes aim to improve the overall user experience and functionality of the application. Please review and merge as appropriate.
![Screenshot (13)](https://github.com/arghadipmanna101/Flipkart_Clone/assets/129872752/0e9e60d2-75cd-436f-96f3-36c342639c84)
![Screenshot (14)](https://github.com/arghadipmanna101/Flipkart_Clone/assets/129872752/9119ce0a-0fdd-4948-b14b-07e004b4baeb)

arghadipmanna101 commented:
There have a conflict issue. check it once then recreate a new PR.

Aradhya-005 commented:
Can you please tell me . How do I resolve this issue

Aradhya-005 commented:
I think resolve the issues. Please review and merge as appropriate.

arghadipmanna101 commented:
> I think , i resolve the issues. Please review and merge as appropriate.

Still there have some conflict issue check it once and then recreate a new PR. ok

arghadipmanna101 commented:
something is  wrong. check it once. it is not working properly .

Aradhya-005 commented:
Actually , i am not getting it properly . Can you please tell me , why this is issue is arising and how do i solve it

arghadipmanna101 commented:
> Actually , i am not getting it properly . Can you please tell me , why this is issue is arising and how do i solve it

what changes you done?

Aradhya-005 commented:
After the changes , site looks like this
![Screenshot (13)](https://github.com/arghadipmanna101/Flipkart_Clone/assets/129872752/4aa10315-5179-43b0-b2bc-1d54c1985142)
![Screenshot (14)](https://github.com/arghadipmanna101/Flipkart_Clone/assets/129872752/394343b8-a74f-4b14-96c5-a7981bb86a5a)

in navbar , i changed the size of searchbar and using margin and padding , i aligned it properly . and after that in become a seller part , i have a created a form page which takes the basic data of seller . 
and in footer , changed the layout of social media link . I mean before , they were displaying horizontal  now they are in vertical form .
